examples/service-proxy.yaml

```diff
@@ -1,4 +1,10 @@
-version: '3'
+version: '3.7'
+
+configs:
+  caddy-basic-content:
+    file: ./Caddyfile
+    labels:
+        caddy:
```

Comment:
If the config have the label prefix, we include it's content on the top of the generated caddyfile. Right after warnings comments.